Plugin/Catalog/Block/AbstractBlock.php

```diff
@@ -91,7 +85,6 @@ abstract class AbstractBlock extends Template
      * @param NostoHelperAccount $nostoHelperAccount
      * @param NostoHelperScope $nostoHelperScope
      * @param StateAwareCategoryServiceInterface $categoryService
-     * @param SearchEngine $searchEngineHelper
      * @param Logger $logger
      */
     public function __construct(
```

Comment:
Delete this class. It's not extended by any other class anymore

Reply:
Deleted